tensorboard/webapp/metrics/views/card_renderer/scalar_card_container.ts

```diff
@@ -428,7 +428,10 @@ export class ScalarCardContainer implements CardRenderer, OnInit, OnDestroy {
           headers.push(ColumnHeaders.RELATIVE_TIME);
         } else {
           headers.push(ColumnHeaders.RUN);
+          headers.push(ColumnHeaders.MIN_VALUE);
```

Comment:
I would be nice to have a comment saying this else block is for range selection.

Reply:
Done.